apps/application/flow/step_node/document_extract_node/impl/base_document_extract_node.py

```diff
@@ -70,7 +70,7 @@ def save_image(image_list):
                     # 回到文件头
                     buffer.seek(0)
                     file_content = split_handle.get_content(buffer, save_image)
-                    content.append('## ' + doc['name'] + '\n' + file_content)
+                    content.append('### ' + doc['name'] + '\n' + file_content)
                     break
 
         return NodeResult({'content': splitter.join(content)}, {})
```

Comment:
The code is mostly correct but it can be improved for more readability and possibly for better error handling. Here's a revised version:

```python
def save_image(image_list):
    buffer.seek(0)
    content = []
    
    for doc in image_list:
        try:
            file_content = split_handle.get_content(buffer, save_image)
            content.append(f"### {doc['name']}\n{file_content}")
        except Exception as e:
            print(f"Error processing document {doc['name']}:", e)
    
    return NodeResult({'content': splitter.join(content)}, {})
```

### Corrections and Suggestions:
1. **Use `f-string` for formatting**: This makes the code more readable than concatenating strings with `'+'`.
   
2. **Error Handling**: Added a `try-except` block to catch any exceptions that might occur when processing each document. This helps in logging errors without breaking the execution of other documents.

3. **Improved Error Message**: Changed the error message to include both the problematic document name and the specific exception occurred. This provides context for debugging.

These changes make the code cleaner and more robust.